pypeit/spectrographs/keck_esi.py

```diff
@@ -0,0 +1,404 @@
+"""
+Module for Keck/ESI specific methods.
+
+.. include:: ../include/links.rst
+"""
+import os
+
+from IPython import embed
+
+import numpy as np
+
+from astropy.time import Time
+
+from pypeit import msgs
+from pypeit import telescopes
+from pypeit import io
+from pypeit.core import framematch
+from pypeit.core import parse
+from pypeit.spectrographs import spectrograph
+from pypeit.images import detector_container
+
+class KeckESISpectrograph(spectrograph.Spectrograph):
+    """
+    Child to handle Keck/ESI specific code
+    """
+    ndet = 1
+    name = 'keck_esi'
+    camera = 'ESI'
+    header_name = 'ESI'
+    #url = 'https://www.lco.cl/?epkb_post_type_1=mage'
+    ech_fixed_format = True
+    telescope = telescopes.KeckTelescopePar()
+    pypeline = 'Echelle'
+    supported = True
+    comment = 'See :doc:`mage`'
```

Comment:
remove this one until we make a ESI doc page

Reply:
gone